fivox/attenuationCurve.h

```diff
@@ -29,6 +29,7 @@
 
 namespace fivox
 {
+
 /** Attentuation curve access for depth-based interpolation. @sa VSDLoader */
 class AttenuationCurve
```

Comment:
I'd rather keep is at Attenuation, at least for now. It is only used for that; the input file containing the values has a custom specific format; and changes/additions attenuation-specific might be done, based on user feedback.